browser/brave_wallet/brave_wallet_event_emitter_browsertest.cc

```diff
@@ -134,7 +135,7 @@ IN_PROC_BROWSER_TEST_F(BraveWalletEventEmitterTest,
       browser()->tab_strip_model()->GetActiveWebContents();
   WaitForLoadStop(contents);
   auto controller = GetEthJsonRpcController();
-  controller->SetNetwork(brave_wallet::mojom::Network::Goerli);
+  controller->SetNetwork(brave_wallet::GetAllKnownNetworks()[3].chain_id);
```

Comment:
Better to find the chain you're looking for instead of an index here because the list can change in the future and the test might not check what it used to check.

Reply:
Added precompiled chain ids to mojom file as @yrliou proposed above